DemoPage/examples/PageContent/index.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+import React from 'react'
+import PageContent from '../../../layout/PageContent'
+import AsyncAction from '../../../buttons/AsyncActionToggle'
+import Button from '../../../buttons/Button'
+import LeadCopy from '../../../typography/LeadCopy'
+import sync from '../../../lib/sync'
+
+export default React.createClass({
+  displayName: 'PageContentExample',
+
+  unsubscribe(bool) {
+    let api = 'https://everydayhero-staging.com/api/v2/notification_options'
+    let namespace = 'page_received_donation'
+    let token = '62d35e46ee366be584e7d6529dd22077c021677d2efe21f03739dd30b39d3723'
+    let url = `${ api }/${ namespace }?access_token=${ token }&value=${ bool }`
```

Comment:
It's neither here nor there for that: that's just the demo file. The real implementation will be in supporter so it won't even have es6 const/let. 

In terms of my opinion on such, I'm inclined towards convention of only using const where you would define it as "var NAMESPACE" in es5, and "let" everywhere else. But I can appreciate the explicitness of using const instead.